SU2_CFD/include/output/tools/CWindowingTools.hpp

```diff
@@ -0,0 +1,120 @@
+/*!
+ * \file signal_processing_toolbox.hpp
+ * \brief Header file for the signal processing toolbox.
+ * \author T. Albring
+ * \version 7.0.0 "Blackbird"
+ *
+ * SU2 Project Website: https://su2code.github.io
+ *
+ * The SU2 Project is maintained by the SU2 Foundation 
+ * (http://su2foundation.org)
+ *
+ * Copyright 2012-2019, SU2 Contributors (cf. AUTHORS.md)
+ *
+ * SU2 is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * SU2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with SU2. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <vector>
+#include "../../../Common/include/option_structure.hpp"
+
+class CWindowingTools{
+public:
+  /*! \brief Returns the value of a windowing function given by fctIdx at time-step CurTimeIdx with given time-frame endTimeIdx (i.e. endTimeIdx=nTimeIter, if one starts  windowing at time t =0.)
+   * \param windowId - enum specifing the used window
+   * \param CurTimeIdx - Current time iteration of the solver
+   * \param endTimeIdx - Number of time steps to average over
+   * \return Value of the window-weight-function at time CurTimeIdx with time-frame endTimeIdx
+   */
+  su2double GetWndWeight(WINDOW_FUNCTION windowId, unsigned long CurTimeIdx, unsigned long endTimeIdx) const;
+
+protected:
+  // Long time windows
+  /*! \brief Returns the value of the Hann-window function at time-step i with given end-time endTimeIdx.
+  * \param i - Current time iteration of the solver
+  * \param endTimeIdx - Number of time steps to average over
+  * \return Value of the window-weight-function at time CurTimeIdx with end-time endTimeIdx
+  */
+  su2double HannWindow(unsigned long i, unsigned long endTimeIdx) const;
```

Comment:
As far as I understand you are using unsigned long `i` `CurTimeIdx` and `currentIter` in this file which all have the same meaning : `Current time iteration of the solver`
For consistency it would be nice to choose one for your implementation, preferably `CurTimeIdx` or something new `CurrTimeIter`

Reply:
I replaced the variable-names in questions with `curTimeIter`.